### PR TITLE
fix attemp

### PR DESCRIPTION
**Pull request by zhuoyuT**
Fixes #0000 

- [ ] I have read and agree to the [Code of Conduct](CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](CONTRIBUTING.md).

## Pull Request Type

- [ ] Bug fix (please link related issues if applicable)
- [ ] New feature
- [ ] Question update
- [ ] Other

### CODE REVIEW

**Comment by danieldanielecki** [comment]
@eduardconstantin, what do you think?

The user has reported a bug via Reddit, @zhuoyuT might you confirm that's what is the PR about?

Screenshot 2024-09-11 at 10 25 13

**Comment by eduardconstantin** [comment]
someone else implemented the scoring, I believe this happens because some questions have multiple answers and the score is divided by the number of answers

**Comment by zhuoyuT** [comment]
@danieldanielecki 
Hey it is, but this is not final. I commented out the shuffle function and reduced the question count for debugging purpose, I will try to update it during the weekend, but feel free to adjust on top of my commit.

@eduardconstantin 
the issue I encountered was the point of the last question was never added

**Comment by danieldanielecki** [comment]
> @danieldanielecki Hey it is, but this is not final. I commented out the shuffle function and reduced the question count for debugging purpose, I will try to update it during the weekend, but feel free to adjust on top of my commit.
> 
> @eduardconstantin the issue I encountered was the point of the last question was never added

thanks for the clarification, will be waiting for the next commits :)

**Comment by zhuoyuT** [comment]
@eduardconstantin @danieldanielecki PR updated, please check

**Comment by danieldanielecki** [comment]
> @eduardconstantin @danieldanielecki PR updated, please check

Is it now 100% fixed & ready?

**Comment by danieldanielecki** [comment]
What I understood before this fix when there was a question with multiple answers correct, the total result wasn't calculated correctly. Now it should be fixed, and easily noticed by seeing 100% for such tests in Practice Mode.

Might you please confirm?

cc: @eduardconstantin

**Comment by eduardconstantin** [comment]
> What I understood before this fix when there was a question with multiple answers correct, the total result wasn't calculated correctly. Now it should be fixed, and easily noticed by seeing 100% for such tests in Practice Mode.
> 
> Might you please confirm?
> 
> cc: @eduardconstantin

I assumed that was the issue, I haven't checked

**Comment by danieldanielecki** [comment]
and it's no longer the case, thank you so much for the contribution @zhuoyuT 

Screenshot 2024-10-03 at 10 23 02

@eduardconstantin, I also made the counter of answered questions in the top left a bit more explicit by adding `ANSWERED: `. Thank you as well, for the input!

File: app/modes/page.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const Modes: NextPage<{ searchParams: { url: string; name: string } }> = ({
         <ExamLink
           href={{
             pathname: "/practice",
-            query: { url },
+            query: { url, name },
```
Comment (danieldanielecki) [inline app/modes/page.tsx:22]: thanks for aligning it with the root page query

**Comment by eduardconstantin** [comment]
> and it's no longer the case, thank you so much for the contribution @zhuoyuT
> 
> Screenshot 2024-10-03 at 10 23 02
> @eduardconstantin, I also made the counter of answered questions in the top left a bit more explicit by adding `ANSWERED: `. Thank you as well, for the input!

not a bad idea, but now the top part is not centered anymore :(

**Comment by danieldanielecki** [comment]
> > and it's no longer the case, thank you so much for the contribution @zhuoyuT
> > Screenshot 2024-10-03 at 10 23 02
> > @eduardconstantin, I also made the counter of answered questions in the top left a bit more explicit by adding `ANSWERED: `. Thank you as well, for the input!
> 
> not a bad idea, but now the top part is not centered anymore :(

uhh good feedback, missed it. Any idea how to handle both?

**Comment by danieldanielecki** [comment]
> > and it's no longer the case, thank you so much for the contribution @zhuoyuT
> > Screenshot 2024-10-03 at 10 23 02
> > @eduardconstantin, I also made the counter of answered questions in the top left a bit more explicit by adding `ANSWERED: `. Thank you as well, for the input!
> 
> not a bad idea, but now the top part is not centered anymore :(

used emoji instead https://github.com/Ditectrev/Practice-Exams-Platform/commit/5ac85c3817f1466e6997898e315455e2e1caade9 and now having best of both ;)

**Comment by danieldanielecki** [comment]
> What I understood before this fix when there was a question with multiple answers correct, the total result wasn't calculated correctly. Now it should be fixed, and easily noticed by seeing 100% for such tests in Practice Mode.
> 
> Might you please confirm?
> 
> cc: @eduardconstantin

I meant Exam Mode of course. Practice Mode has no results shown.

**Pull request merged**